eth_account/signers/local.py

```diff
@@ -86,6 +86,7 @@ def signTransaction(self, transaction_dict):
         warnings.warn(
             "signTransaction is deprecated in favor of sign_transaction",
             category=DeprecationWarning,
+            stacklevel=2,
```

Comment:
I'm curious about this change - is it a `flake8-bugbear` catch?

Reply:
It is `flake8-bugbear`. The full error was:

```
 B028 No explicit stacklevel keyword argument found. The warn method from the warnings module uses a stacklevel of 1 by default. This will only show a stack trace for the line on which the warn method is called. It is therefore recommended to use a stacklevel of 2 or greater to provide more information to the user.
```

It sounded reasonable to me so just added the `stacklevel` arg.

Reply:
Yep, makes sense to me! Thanks!